README.md

```diff
@@ -5,6 +5,55 @@ Terraform configuration files for the Sentieon software
 
 [Terraform](https://www.terraform.io/) is an open-source infrastructure as code (IaC) tool for the provisioning and management of cloud infrastructure. This repository contains example terraform configuration files that can be used to quickly deploy the Sentieon software to your cloud infrastructure.
 
+## Quick Start - Sentieon License server deployment to Azure
+
+### Requirements
+
+* The [Terraform CLI](https://developer.hashicorp.com/terraform/downloads)
+* The [Azure CLI]( https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
+* An Azure account and credentials with permission to provision resources inside the account
+* A Sentieon license file for your FQDN, bound to port 8990 and placed in your Blob storage container
```

Comment:
After ssh-ing into the vm, will the user have permission to access the license file in blob storage?

If we are already relying on ssh to manage the instance, the user could just use scp to upload the license file to eliminate any permissions issues.

Reply:
that is true. They will have a ssh key to upload the license file